Cargo.toml

```diff
@@ -4,7 +4,11 @@ members = [
     "mirrord-protocol",
     "mirrord-agent",
     "mirrord-layer",
-    "mirrord-cli",
+    "mirrord-cli",    
+]
+
+exclude = [
```

Comment:
i am doing this because one of the tests, i believe test_agent_image fails and iirc it was something related to manifest not found. I can rerun the tests with tests being a part of the workspace members and ping you if that error happens again